.gitmodules

```diff
@@ -8,3 +8,6 @@
 [submodule "petsc"]
     path = petsc
     url = https://gitlab.com/petsc/petsc.git
+[submodule "modules/fluid_properties/saline_moose"]
+	path = modules/fluid_properties/saline_moose
+	url = git@code.ornl.gov:neams/saline_moose.git
```

Comment:
No, let's just directly include the `saline` repo as a submodule and let's get rid of the `saline_moose` repo alltogether.

Reply:
I am fine with that but will need guidance on how to get the `saline` repo working with the MOOSE build system.  That was my reason for repackaging it into `saline_moose` in the first place.

Reply:
Yeah, you can look at my Thermocimica PR [here](https://github.com/idaholab/moose/pull/22712).

Reply:
More specifically look at the [`thermochimica.mk`](https://github.com/idaholab/moose/pull/22712/files#diff-3c66f7d8738957757c65dc1e904ff253939e6b13a7f9834625b030a30f9c6f48) file (and where include it from [`chemical_reactions.mk`](modules/chemical_reactions/chemical_reactions.mk)).

On that note it would be cool if you could retain the same directory layout and have the saline submodule under `modules/fluid_properties/contrib/saline`.

Reply:
with the build magic located in `modules/fluid_properties/contrib/saline.mk`

Reply:
Should be fixed now